package.json

```diff
@@ -13,13 +13,13 @@
   },
   "devDependencies": {
     "@vitejs/plugin-react": "4.3.1",
-    "@vitest/coverage-v8": "^1.6.0",
-    "@vitest/ui": "1.6.0",
-    "prettier": "^3.3.2",
+    "@vitest/coverage-v8": "^2.0.3",
```

Comment:
just to be sure and since those are major version changes, can you please double check the license type didn't change ?